hedera-node/src/main/java/com/hedera/services/txns/schedule/ScheduleCreateTransitionLogic.java

```diff
@@ -86,13 +86,15 @@ public ResponseCodeEnum validate(TransactionBody txnBody) {
             return validity;
         }
 
-        var signers = op.getSigners().getAccountsList();
-        if (signers.isEmpty()) {
-            return EMPTY_SIGNERS_LIST;
+        if (!op.hasPayer()) {
+            return MISSING_SCHEDULE_PAYER;
         }
-        if (signers.size() < op.getSigners().getThreshold()) {
-            return INVALID_SCHEDULE_THRESHOLD;
+
+        var sigMap = op.getSigMap();
+        if (op.hasSigMap() || sigMap.getSigPairList().isEmpty()) {
+            return EMPTY_SIGNATURES;
```

Comment:
sigMap is optional. It is stated in the comment describing it. No need to check it